src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -745,11 +745,6 @@ void SurgeGUIEditor::refresh_mod()
    }
    // ctnvg	frame->redraw();
    // frame->setDirty();
-#if LINUX
-   frame->invalid();
-   // Turns out linux is very bad with lots of little invalid rects in vstgui
-   // see github issue 1103
-#endif
```

Comment:
Why did you remove this? It’s an important Linux optimization!

Reply:
Looks like you weren’t against master. Pull master, check out this branch again, do a “git rebase master” then force push and you should be ok. But need to do that before I merge obvs

Reply:
Yeah not sure how I ended up against master... reinstated now!